airgun/views/oscaptailoringfile.py

```diff
@@ -14,17 +14,19 @@
 from airgun.widgets import (
     ActionsDropdown,
     MultiSelect,
-    SatTable,
+    Table,
 )
 
 
 class SCAPTailoringFilesView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Tailoring Files']")
     new = Text("//a[contains(@href, 'tailoring_files/new')]")
-    table = SatTable(
+    table = Table(
         './/table',
         column_widgets={
-            'Title': Text('./a'),
+            'Name': Text("./a"),
+            'Filename': Text("./a"),
+            'Created at': Text("./a"),
```

Comment:
Changing `Title` to `Name` is good.

The other two are incorrect. Widget definition is created relative to cell element in table body, not in header, and as far as I can see, these cells contain plain text, not links. So these widgets won't match anything.

Also, `column_widgets` only has to contain definitions for cells with widgets that widgetastic can't figure on it's own. You can, and should, skip columns that contain only text, or with which you don't plan to interact in any away.
Case in point: in current master, `view.table.read()` returns following list on my toy Satellite:
```python
[
    {'Name': 'whatever', 'Filename': 'ssg-rhel7-ds-tailoring.xml', 'Created at': 'about 1 hour ago', 'Actions': ['Download']},
    {'Name': 'tKNrMxynYL', 'Filename': 'ssg-rhel7-ds-tailoring.xml', 'Created at': 'less than a minute ago', 'Actions': ['Download']}
]
```
As you can see, `Filename` and `Created at` fields are both present, despite not being explicitly mentioned in table definition on Airgun side.

Reply:
Thanks for explaining, I didn't knew that :)
will update PR accordingly.